### PR TITLE
[internal-only] 4-16-rn-doc-tidy: Clean up of 4.16 release notes

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.16

[Link to docs preview](https://78011--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html)

### CODE REVIEW

**Review by bergerhoffer — COMMENTED** [review]
Nice catches throughout! Just questioning a few of them

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -131,15 +130,15 @@ You can define tags for compute or control plane machines on an existing cluster
 For more information, see "Adding tags to machines by using machine sets" for xref:../machine_management/creating_machinesets/creating-machineset-vsphere.adoc#machine-api-vmw-add-tags_creating-machineset-vsphere[compute] or xref:../machine_management/control_plane_machine_management/cpmso_provider_configurations/cpmso-config-options-vsphere.adoc#machine-api-vmw-add-tags_cpmso-config-options-vsphere[control plane] machine sets.
 
 [id="ocp-4-16-admin-ack-updating_{context}"]
-==== Required administrator acknowledgment when updating from {product-title} 4.15 to 4.16
+==== Required administrator acknowledgment when updating from {product-title} 4.15 to {product-version}
 
-{product-title} 4.16 uses Kubernetes 1.29, which removed several xref:../release_notes/ocp-4-16-release-notes.adoc#ocp-4-16-removed-kube-1-29-apis_{context}[deprecated APIs].
+{product-title} {product-version} uses Kubernetes 1.29, which removed several xref:../release_notes/ocp-4-16-release-notes.adoc#ocp-4-16-removed-kube-1-29-apis_{context}[deprecated APIs].
```
[the remark this answers is not in the document]
Comment (dfitzmau) [reply release_notes/ocp-4-16-release-notes.adoc:135]: Thanks. That was my intention going in but these situations were a bit trickier. However, as these notes are 4.16-specific, I changed them.

File: release_notes/ocp-4-16-release-notes.adoc [hunk 2]
```diff
@@ -777,26 +772,30 @@ For further information, see xref:../networking/hardware_networks/configuring-sr
...
 
 For more information about this feature on bare-metal infrastructure, see xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#nw-osp-services-external-load-balancer_ipi-install-installation-workflow[Services for a user-managed load balancer] in _Setting up the environment for an OpenShift installation_.
 
 [id="ocp-4-16-detect-warning-for-iptables_{context}"]
 ==== Detect and warning for iptables
 With this release, if you have pods in your cluster using `iptables` rules the following event message is given to warn against future deprecation:
+
+[source,terminal]
+----
 `This pod appears to have created one or more iptables rules. IPTables is deprecated and will no longer be available in RHEL 10 and later. You should consider migrating to another API such as nftables or eBPF.`
```
[the remark this answers is not in the document]
Comment (dfitzmau) [reply release_notes/ocp-4-16-release-notes.adoc:797]: :heavy_check_mark:

File: release_notes/ocp-4-16-release-notes.adoc [hunk 3]
```diff
@@ -1035,12 +1036,12 @@ By default, when you make certain changes to the parameters in a `MachineConfig`
...
 
 [id="ocp-4-16-machine-config-operator-boot-image_{context}"]
 ==== Updating boot images (Technology Preview)
 
-By default, the MCO does not delete the boot image it uses to bring up a {op-system-first} node. Consequently, the boot image in your cluster is not updated along with your cluster. You can now configure your cluster to update the boot image whenever you update your cluster. For more information, see xref:../post_installation_configuration/machine-configuration-tasks.adoc#mco-update-boot-images_post-install-machine-configuration-tasks[Updating boot images].
+By default, the MCO does not delete the boot image it uses to bring a {op-system-first} node. Consequently, the boot image in your cluster is not updated along with your cluster. You can now configure your cluster to update the boot image whenever you update your cluster. For more information, see xref:../post_installation_configuration/machine-configuration-tasks.adoc#mco-update-boot-images_post-install-machine-configuration-tasks[Updating boot images].
```
[the remark this answers is not in the document]
Comment (dfitzmau) [reply release_notes/ocp-4-16-release-notes.adoc:1044]: OK. Vale flagged the word "up" here but I can revert

File: release_notes/ocp-4-16-release-notes.adoc [hunk 4]
```diff
@@ -1318,36 +1313,35 @@ With this release, `dasd` image artifacts for the `s390x` architecture are remov
 
 [discrete]
 [id="egress-ip-etp-local-support_{context}"]
-=== Support for EgressIP with ExternalTrafficPolicy=Local services
+=== Support for EgressIP with `ExternalTrafficPolicy=Local` services
```
[the remark this answers is not in the document]
Comment (dfitzmau) [reply release_notes/ocp-4-16-release-notes.adoc:1316]: OK. I changed about 6 heading entries to reflect this.

File: release_notes/ocp-4-16-release-notes.adoc [hunk 5]
```diff
@@ -1318,36 +1313,35 @@ With this release, `dasd` image artifacts for the `s390x` architecture are remov
...
 
-Previously, it was unsupported for EgressIP selected pods to also serve as backends for services with `externalTrafficPolicy` set to `Local`. When attempting this configuration, service ingress traffic reaching the pods was incorrectly rerouted to the egress node hosting the EgressIP. This affected how responses to incoming service traffic connections were handled and led to non-functional services when `externalTrafficPolicy` was set to `Local`, as connections were dropped and the service became unavailable.
+Previously, this feature was unsupported for EgressIP selected pods to also serve as backends for services with `externalTrafficPolicy` set to `Local`. When attempting this configuration, service ingress traffic reaching the pods was incorrectly rerouted to the egress node hosting the EgressIP. This affected how responses to incoming service traffic connections were handled and led to non-functional services when `externalTrafficPolicy` was set to `Local`, as connections were dropped and the service became unavailable.
```
[the remark this answers is not in the document]
Comment (dfitzmau) [reply release_notes/ocp-4-16-release-notes.adoc:1318]: NP. "It" is a vague referrent in itself but I'll keep as I'm not knwoledgeable with these components.

File: release_notes/ocp-4-16-release-notes.adoc [hunk 6]
```diff
@@ -1318,36 +1313,35 @@ With this release, `dasd` image artifacts for the `s390x` architecture are remov
...
-In this release, the functionality to authenticate to private registries on {aws-first}, {gcp-first}, and {azure-first} clusters is moved from the in-tree provider to binaries that ship with {product-title}.
-This change supports the default external cloud authentication provider behavior that is introduced in Kubernetes 1.29.
+In this release, the functionality to authenticate to private registries on {aws-first}, {gcp-first}, and {azure-first} clusters is moved from the in-tree provider to binaries that ship with {product-title}. This change supports the default external cloud authentication provider behavior that is introduced in Kubernetes 1.29.
 
 [discrete]
 [id="ocp-4-16-olmv1-default-upgrade-constraints_{context}"]
 === Default {olmv1} upgrade constraints changed to {olmv0} semantics (Technology Preview)
 
-In {product-title} 4.16, {olmv1-first} changes its default upgrade constraints from semantic versioning (semver) to {olmv0} semantics.
+In {product-title} {product-version}, {olmv1-first} changes its default upgrade constraints from semantic versioning (`semver`) to {olmv0} semantics.
```
[the remark this answers is not in the document]
Comment (dfitzmau) [reply release_notes/ocp-4-16-release-notes.adoc:1356]: Confirmed that backticks are not required. Reverted.

File: release_notes/ocp-4-16-release-notes.adoc [hunk 7]
```diff
@@ -1010,7 +1011,7 @@ For more information, see xref:../operators/olm_v1/olmv1-installing-an-operator-
...
 
-There are use cases where you might want to allow unauthenticated users the `system:webhook` role binding for specific namespaces. The `system:webhook` cluster role allows users to trigger builds from external systems that do not use {product-title} authentication  mechanisms, such as GitHub, GitLab, and Bitbucket. Cluster admins can grant unauthenticated users access to the `system:webhook` role binding to facilitate this use case.
+There are use cases where you might want to allow unauthenticated users the `system:webhook` role binding for specific namespaces. A user with the `system:webhook` cluster role can trigger builds from external systems that do not use {product-title} authentication  mechanisms, such as GitHub, GitLab, and Bitbucket. Cluster administrators can grant unauthenticated users access to the `system:webhook` role binding to provide this use case.
```
[the remark this answers is not in the document]
Comment (dfitzmau) [reply release_notes/ocp-4-16-release-notes.adoc:1026]: Checked with Courtney. Reverted to original text.

**Comment by ShaunaDiaz** [comment]
Taking out of the merge review queue because there is a conflict.

**Comment by dfitzmau** [comment]
> Taking out of the merge review queue because there is a conflict.

Yeah, this happened outside my timezone so I'm going to push to get this merged during my time online.

**Pull request merged**